internal/main.go

```diff
@@ -112,8 +112,8 @@ func main() {
 		// translate from 2 to 3
 		mapping := getMapping(fsMap)
 
-		// parse to 2.4.0-experimental
-		cfg, rpt, err := v2_4_experimental.Parse(dataIn)
+		// parse
+		cfg, rpt, err := v2_3.Parse(dataIn)
```

Comment:
This technically is a potentially breaking change if someone was using a field that is present in `2.4.0-experimental` but not `2.3.0`; might make sense to switch to either `2.4.0` or `2.5.0-experimental` (with Ignition `0.35.0`)

Reply:
I dont think this is a breaking change bc main.go in master currently does not work at all.

It gives error:
`internal/main.go:122:35: cannot use cfg (type "github.com/coreos/ignition/config/v2_4_experimental/types".Config) as type "github.com/coreos/ignition/config/v2_3/types".Config in argument to ign2to3.Translate`

With my change it actually works now.

Reply:
Ah, I see now that the actual `Translate` function is expecting a `v2_3` config.